app/services/remediations_api.rb

```diff
@@ -3,15 +3,15 @@
 # This class is meant to contain all calls to the Remediations API.
 class RemediationsAPI
   def initialize(account)
-    @url = URI.parse("#{URI.parse(Settings.host_inventory_url)}"\
+    @url = URI.parse("#{URI.parse(Settings.remediations_url)}"\
```

Comment:
The remediations URL cannot be created using `Settings.host_inventory_url` in any environment other than production